InputHelpers/FileInputHelper.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.IO;
+using System.Xml.Linq;
+using Newtonsoft.Json.Linq;
+namespace InputHelpers
+{
+    public static class FileInputHelper
+    {
+        public static string ReadJsonFile(string message)
+        {
+            while (true)
+            {
+                string filePath = ReadInputHelper.GetStringInput(message);
+
+                if (IsFilePathValidJson(filePath))
+                {
+                    return filePath;
+                }
+                else
+                {
+                    Console.WriteLine("Invalid JSON file path or format. Please enter a valid JSON file path.");
+                }
+            }
+        }
+
+        private static bool IsFilePathValidJson(string filePath)
```

Comment:
I didn't like this, the parser it self should check if the passed string/file is in the valid format or not.